### ISSUE TITLE
Error: 'Characters' isn't a type.   final _textCharacters = <Characters>[];

### ISSUE THREAD

**Issue by namanawasthi** [aagarwal1012/Animated-Text-Kit#151]
**Description**
I am getting below error due to which i am unable to build my gradle. Seems there is some issue with the package because on removing the Animated text kit package gradle is build successfully.

Error:-
Compiler message:
/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/animated_text_kit-2.5.4/lib/src/colorize.dart:109:28: Error: 'Characters' isn't a type.
  final _textCharacters = <Characters>[];
                           ^^^^^^^^^^

**Piece of code where Animated text kit package is used**
TypewriterAnimatedTextKit(
                    text: ["Cleaning Cowboys"],
                    textStyle: TextStyle(
                      fontSize: 35.0,
                      fontWeight: FontWeight.w300,
                      color: Colors.black,
                    ),
                  ),

**Expected behavior**
There should be no such error and build should be successful.

**Screenshots**
![image](https://user-images.githubusercontent.com/32276228/99501188-6ff91380-29a1-11eb-879a-9091ad0dc562.png)

**Flutter:**
Flutter 1.17.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 8af6b2f038 (5 months ago) • 2020-06-30 12:53:55 -0700
Engine • revision ee76268252
Tools • Dart 2.8.4

**Additional Points**
I Tried "Flutter Clean" , "Flutter Pub Get" and "Flutter Run but the issue is not resolved. Only after removing the Animated kit package it is working.  But i don't understand the root cause of it because earlier it was working with Animated Kit Package. Please help me here.

**Comment by aagarwal1012** [comment]
Duplicate of #140

**Comment by aagarwal1012** [comment]
@namanawasthi, please check the solution mentioned in #140 to solve your issue.

**Comment by awhitford** [comment]
It looks like we are tracking this under #141 now.
